.github/workflows/dart_ci.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        sdk: [ 2.18.7, stable, beta ]
+        sdk: [ 2.18.7 ]
```

Comment:
Note Dart 3+ was removed because everything 2.19 and greater fail to build with an older analyzer version than 5. 
Underlying root cause https://github.com/dart-lang/sdk/issues/51128
Newer Dart versions are added back in the PR that updates to analyzer 5 here https://github.com/davidmarne/built_redux/pull/171/files#diff-0530c3bf28665689d37061f6d19aade372700afec3ae45decc3f2edbd64e344dR19